src/plugins/intel_gpu/src/runtime/ocl/ocl_event.cpp

```diff
@@ -103,6 +104,32 @@ bool ocl_event::get_profiling_info_impl(std::list<instrumentation::profiling_int
     return true;
 }
 
+std::pair<uint64_t, uint64_t> ocl_event::get_host_timestamps(const stream& s) const {
```

Comment:
How about adding `get_host_timestamps()` implementation to `ocl_events` as well?